.github/workflows/build.yml

```
@@ -0,0 +1,32 @@
+name: Build Negativity
+
+# For each push on all branchs
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+    - name: Set up Java
+      uses: actions/setup-java@v3
+      with:
+        distribution: 'temurin'
+        java-version: 17
+        cache: 'gradle'
+    - uses: actions/cache@v3
+      with:
+        path: |
+          ~/.gradle/caches
+          ~/.gradle/wrapper
+          ~/.m2
+        key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties') }}
+        restore-keys: |
+          ${{ runner.os }}-gradle-
+    - name: Grant execute permission for all runners
+      run: chmod +x gradlew setup_spigot.sh
+    - name: Run setup spigot script
+      run: bash setup_spigot.sh
+    - name: Build with Gradle
+      run: ./gradlew shadowJar
```

Une raison pour ne pas utiliser build ?
On rate les tests et autres (futurs ?) checks en utilisant shadowJar directement

```suggestion
      run: ./gradlew build
```